src/BenchmarkDotNet.Core/Attributes/Jobs/UapJobAttribute.cs

```diff
@@ -0,0 +1,18 @@
+﻿using BenchmarkDotNet.Environments;
+using BenchmarkDotNet.Jobs;
+
+namespace BenchmarkDotNet.Attributes.Jobs
+{
+    public class UapJobAttribute : JobConfigBaseAttribute
+    {
+        public UapJobAttribute(string devicePortalUri, string csfrCookie, string wmidCookie, string uapBinariesPath)
```

Comment:
ok, so could you put a link to the documentation in the summary header? You have done a great job and if you  want other people to use 100% of the feature you implemented you need to let them know how to use it properly